style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] By default, flexbox is in column direction, I guess you don't need this line of codes, you can delete them.